src/mixins/PredicateClient.sol

```diff
@@ -2,11 +2,10 @@
 
 pragma solidity ^0.8.12;
 
-import {Ownable} from "openzeppelin/access/Ownable.sol";
 import {IServiceManager, Task} from "../interfaces/IServiceManager.sol";
 import {IPredicateClient, PredicateMessage} from "../interfaces/IPredicateClient.sol";
 
-contract PredicateClient is IPredicateClient, Ownable {
+abstract contract PredicateClient is IPredicateClient {
```

Comment:
not a problem, but this does add a bit of work for integration

Reply:
yes